JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1427,4 +1437,82 @@ Descriptives <- function(jaspResults, dataset, options) {
     return(TRUE)
   else
     return(FALSE)
-}
\ No newline at end of file
+}
+
+.XbarchartNoId <- function(jaspResults, dataset, options) {
+
+  ready <- (length(options$variables) > 1)
+  if (!ready)
+    return()
+
+  jaspResults[["XbarPlot"]] <- createJaspPlot(title = "X bar chart", width = 650, aspectRatio = 1)
+  jaspResults[["XbarPlot"]]$dependOn(c("Xbarchart", "variables"))
+  jaspResults[["XbarPlot"]]$position <- 11
+  XbarPlot <- jaspResults[["XbarPlot"]]
+
+  data1= na.omit(as.data.frame(dataset))
+  means <- rowMeans(data1)
+  subgroups <- 1:length(means)
+  data2 <- data.frame(subgroups = subgroups, means = means)
+  sixsigma <- qcc::qcc(data1, type = 'xbar', plot = F)
+  center <- sixsigma$center
+  sd1 <- sixsigma$std.dev
+  UCL <- max(sixsigma$limits)
+  LCL <- min(sixsigma$limits)
+
+  p <- ggplot2::ggplot(data2, ggplot2::aes(x = subgroups, y = means, group = 1))+ ggplot2::geom_line()+
```

Comment:
```suggestion
  p <- ggplot2::ggplot(data2, ggplot2::aes(x = subgroups, y = means, group = 1)) + ggplot2::geom_line()+
```
please always add a space before the `+`.